manifests/l/LastFM/LastFMDesktopScrobbler/3.1.29/LastFM.LastFMDesktopScrobbler.locale.en-US.yaml

```diff
@@ -0,0 +1,10 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+PackageIdentifier: LastFM.LastFMDesktopScrobbler
+PackageVersion: 3.1.29
+PackageLocale: en-US
+Publisher: LastFM
```

Comment:
```suggestion
Publisher: Last.fm
```

Reply:
this change will also require changes to PackageIdentifier and folders, right?

Reply:
Nope, the folder structure does not have to match, that would also cause another issue..